shuup/admin/modules/service_providers/views/_wizard.py

```diff
@@ -14,7 +14,7 @@
 
 
 class ServiceProviderTypeForm(forms.Form):
-    providers = forms.CharField(label=_("Provider"), required=False)
+    providers = forms.CharField(label=_("Provider"), required=False, widget=forms.HiddenInput())
```

Comment:
we need to add some margin here:
![image](https://user-images.githubusercontent.com/8770370/45952656-14f44d80-bfde-11e8-9668-9e5bdab84c85.png)